utils/ioc_launcher.py

```diff
@@ -1,10 +1,27 @@
 import subprocess
 import os
+from time import sleep
 
 from utils.channel_access import ChannelAccess
 from utils.log_file import log_filename
 
 
+EPICS_TOP = os.path.join("C:\\", "Instrument", "Apps", "EPICS")
```

Comment:
This is set in an environment variable called KIT_ROOT when running config_env. Might be better to use that?